src/library/scala/ref/ReferenceQueue.scala

```diff
@@ -16,10 +16,11 @@ package scala.ref
  *  @author Sean McDirmid
  *  @author Philipp Haller
  */
+@deprecated("Use `java.lang.ref.ReferenceQueue` instead.",since = "2.13.0-RC1")
 class ReferenceQueue[+T <: AnyRef] {
 
   private[ref] val underlying: java.lang.ref.ReferenceQueue[_ <: T] = new java.lang.ref.ReferenceQueue[T]
-  override def toString = underlying.toString
+  override def toString: String = underlying.toString
```

Comment:
So because it's an override, the annotation is not useful.

Reply:
The typer will infer the type from its parent?

Reply:
There's an issue open for that, apparently it doesn't yet. But for String, the extra chars don't add much. `override def toString = ???` would narrow the type. Not a big deal, I don't know why I mentioned it. Probably I was waiting for a build to finish.